src/App.css

```diff
@@ -113,3 +113,24 @@ li button:hover {
   background-color: var(--button-bg);
   color: white;
 }
+.completion-message {
```

Comment:
I would like the message to show up next to the completed item, not in the top right corner.

Reply:
I would like the message to show up next to the completed item, not in the top right corner.

Reply:
Currently the nice message has a strikethrough through it when it is next to the completed item. Can we fix that?

Reply:
If I complete and uncomplete an item quickly, the 'timeout' for showing the message hides / shows very quickly. Can we make this more robust, so that we 'cancel' the timeout when we uncomplete an item?

Reply:
If I complete and uncomplete an item quickly, the 'timeout' for showing the message hides / shows very quickly. Can we make this more robust, so that we 'cancel' the timeout when we uncomplete an item?

Reply:
If I complete and uncomplete an item quickly, the 'timeout' for showing the message hides / shows very quickly. Can we make this more robust, so that we 'cancel' the timeout when we uncomplete an item?